packages/alfa-css/src/syntax/token.ts

```diff
@@ -1346,16 +1346,14 @@ export namespace Token {
   export const parseCloseComment = parseToken(isCloseComment);
 }
 
-function parseToken<T extends Token>(
-  refinement: Refinement<Token, T>
-): Parser<Slice<Token>, T, string> {
-  return (input) =>
+function parseFirst(): Parser<Slice<Token>, Token, string> {
+  return (input: Slice<Token>) =>
```

Comment:
```suggestion
const parseFirst = (input: Slice<Token>) =>
```
(no need to create an extra continuation)